sdk/go/common/registry/preregistry.go

```diff
@@ -16,23 +16,35 @@ package registry
 
 import "strings"
 
-// PulumiPublishedBeforeRegistry checks if name could be a valid package, even though it
-// hasn't been published in the Pulumi package registry.
+// IsPreGitHubRegistryPackage checks if a package name corresponds to a Pulumi package
+// that was published before the GitHub Public Pulumi Registry existed and should be
```

Comment:
What is the github public registry? The code at `github.com/pulumi/registry` has never played into this. The public registry in question is the unauthenticated component of our API.